database/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const async = require('async')
+const pg = require('pg')
+const config = require('./../src/lib/config')
+
+if (config.get('env') !== 'test') {
```

Comment:
@mihaidma I've used the `LABS_AUTH_SERVICE_env` because I wanted something specific that you change voluntarily if init/loading is what you really want :)

@paolochiodi @mihaidma As of now we do not have separate conf for separate envs, `LABS_AUTH_SERVICE_env` is just a flag, should I change it's name given it's creating confusion?

How about `LABS_AUTH_SERVICE_local=true`?